articles/client-auth/v2/mobile-desktop.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Authentication for Mobile & Desktop Apps
+description: Explains how to authenticate users in a mobile or desktop application.
+toc: true
+---
+
+# Authentication for Mobile & Desktop Apps
+
+You can authenticate users of your mobile/desktop applications by:
+
+* Using the [Lock](/libraries/lock) client libraries;
+* Calling the Auth0 [Authentication API](/api/authentication) endpoints.
```

Comment:
I would add something to clarify the difference between these two options (for the sake of those not familiar with out product). Something simple like: 

```
* Using [Lock](/libraries/lock), a drop-in authentication widget that provides a standard set of behaviors and a customizable user interface, or
* Calling the Auth0 [Authentication API](/api/authentication) endpoints, in case you want to implement your own UI.
```